AIMoba/AIMoba/Controllers/GameController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AIMoba.Controllers
+{
+    public class GameController : Controller
+    {
+        public IActionResult Game()
+        {
+            return View();  //Megjeleníti az oldalt.
+        }
+        
+        public class Player
```

Comment:
Ez miért a GameController osztálynak egy al-osztálya? Számomra már az is meglepő, hogy ez működik :D Java-ban nem is lehet public alosztályt csinálni

Reply:
Kiszedtem a GameController osztályból, először console-ban írtam, és rossz helyre másoltam be.